dev-requirements.txt

```diff
@@ -20,5 +20,5 @@ twine
 
 # Requirements for testing [async] extra
 aiohttp; python_version>="3.6"
-pytest-asyncio<=0.18.3; python_version>="3.6"
+pytest-asyncio<=0.21.0; python_version>="3.6"
```

Comment:
version 0.19 for pytest-asyncio caused failures, we can make sure we avoid that version.

Reply:
Hello @VachaShah, version 0.19 failed before because default mode from the version 0.19 is changed to auto. Now, I mentioned to use mode as auto. So, version 0.19 will not fail. And I left it as <=0.21.0 because pytest-asyncio only supports from python_version = 3.7.